frontend/components/modal/ModalBase.vue

```diff
@@ -1,11 +1,15 @@
 <template>
-  <Dialog @close="closeModal()" class="relative z-50" :open="modalIsOpen">
-    <div
-      @click="closeModal()"
-      class="fixed inset-0 cursor-pointer bg-light-layer-0/95 dark:bg-dark-layer-0/95"
-      aria-hidden="true"
+  <Dialog class="relative z-50" :open="modalIsOpen">
+    <DialogBackdrop
+      className="fixed inset-0 bg-light-layer-0/95 dark:bg-dark-layer-0/95"
     />
+    <!-- There are linter warnings about @click="closeModal()" -->
```

Comment:
With merged in frontend updates I ran the following:

```bash
yarn typecheck
yarn lint
```

... and we don't need these disables anymore 😊 Removing ✅

Reply:
Super! I've made a little gist with these yarn commands, so that I remember what to do when I'm preparing a PR.

Reply:
Is also in the contributing guide now :) You can also check the `package.json` file for the scripts that can be run in the frontend 😊